pom.xml

```diff
@@ -1000,7 +1000,7 @@
             <!--Hamcrest first, then JUnit, then Mockito. See http://goo.gl/e5bJA5-->
             <groupId>org.hamcrest</groupId>
             <artifactId>hamcrest-all</artifactId>
-            <version>1.1</version>
+            <version>1.3</version>
             <scope>test</scope>
         </dependency>
         <dependency>
```

Comment:
Exactly. Because of the challenges coordinating across DDF, DIB and GeoBus, this PR only tackles hamcrest. The PR description calls out Mockito and JUnit, but misses xmlunit. I'll update the description to xmlunit include as well.